pkg/options/options.go

```diff
@@ -108,6 +109,8 @@ func (s *KubeRouterConfig) AddFlags(fs *pflag.FlagSet) {
 		"The graceful period before removing destinations from IPVS services (e.g. '5s', '1m', '2h22m'). Must be greater than 0.")
 	fs.BoolVar(&s.IpvsGracefulTermination, "ipvs-graceful-termination", false,
 		"Enables the experimental IPVS graceful terminaton capability")
+	fs.BoolVar(&s.IpvsPermitAll, "ipvs-permit-all", true,
+		"Enables rule to accept all incoming traffic on the node. Default: false.")
```

Comment:
No need for `Default: false` in the flags comment.  Please remove it.

Can you please rephrase to `Enables rule to accept all incoming traffic to service VIP's on the node`

Reply:
Done.